unstructured/ingest/connector/weaviate.py

```diff
@@ -0,0 +1,134 @@
+import json
+import typing as t
+from dataclasses import dataclass
+
+from unstructured.ingest.interfaces import (
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.utils import requires_dependencies
+
+
+@dataclass
+class SimpleWeaviateConfig(BaseConnectorConfig):
```

Comment:
I found that (for example) pinecone did not allow parallel requests when I tried to upload content in parallel

However, if Weaviate allows it, it might be good to have session_handles here

https://github.com/Unstructured-IO/unstructured/pull/1774/files might help on session handles, if it's relevant